pom.xml

```diff
@@ -5,149 +5,79 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.11</version>
-    <relativePath />
+    <version>3.26</version>
   </parent>
   <groupId>io.snyk.plugins</groupId>
   <artifactId>snyk-security-scanner</artifactId>
   <version>${revision}</version>
   <packaging>hpi</packaging>
 
-  <properties>
-    <jenkins.version>1.625.3</jenkins.version>
-    <java.level>7</java.level>
-    <!-- will be overridden in CD pipeline -->
-    <revision>LOCAL-SNAPSHOT</revision>
-  </properties>
-
   <name>Snyk Security Plugin</name>
   <description>Add the ability to test your code dependencies for vulnerabilities against Snyk database</description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Snyk+Security+Plugin</url>
 
+  <scm>
+    <connection>scm:git:ssh://github.com/jenkinsci/snyk-security-scanner-plugin.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/snyk-security-scanner-plugin.git</developerConnection>
+    <url>https://github.com/jenkinsci/snyk-security-scanner-plugin</url>
+  </scm>
+  <ciManagement>
+    <system>Travis CI</system>
+    <url>https://travis-ci.org/jenkinsci/snyk-security-scanner-plugin</url>
+  </ciManagement>
   <licenses>
     <license>
       <name>MIT License</name>
       <url>http://opensource.org/licenses/MIT</url>
     </license>
   </licenses>
 
-  <repositories>
-    <repository>
-      <id>repo.jenkins-ci.org</id>
-      <url>https://repo.jenkins-ci.org/public/</url>
-    </repository>
-  </repositories>
-  <pluginRepositories>
-    <pluginRepository>
-      <id>repo.jenkins-ci.org</id>
-      <url>https://repo.jenkins-ci.org/public/</url>
-    </pluginRepository>
-  </pluginRepositories>
+  <properties>
+    <jenkins.version>2.0</jenkins.version>
```

Comment:
Bump from pervious `1.625` - what do we gain from this, and what do we lose?

Reply:
This was from our "open questions" (question 6.) from first Google Doc requirement. We can change it (but first I have to check classes, so no new (2.0* jenkins code) used)